ctapipe/resources/train_particle_classifier.yaml

```diff
@@ -6,13 +6,11 @@
 # ======================================================================
 
 TrainParticleClassifier:
-  random_seed: 0              # Seed used for sampling n_* events for training.
-  n_signal:                   # The number of signal events used for training that can be provided
+  random_seed: 0              # Seed used for sampling n_events for training.
+  n_events:                   # The number of events used for training that can be provided
     # - [type, "LST*", 1000]  # independently for each telescope type (e.g. "LST_LST_LSTCam").
-    # - [type, "MST*", 1000]  # If not specified, all events in the file are used.
-  n_background:               # Same as above, but for background events.
-    # - [type, "LST*", 1000]
-    # - [type, "MST*", 1000]
+    # - [type, "MST*", 1000]  # If not specified, as many events as possible are used.
+  signal_fraction: 0.5        # signal_fraction = n_signal / n_events
```

Comment:
I would find it more intuitive I think to define the ratio? 

I.e. 1.0 means use as much signal as background?

Reply:
I think signal fraction being how much signal is in the total is pretty intuitive?

Reply:
I think there are merits to both options:
- signal fraction in combination with `n_events` makes it a little bit more intuitive how many signal and how many background events are used, but we write out this information in the logs anyway.
- signal ratio is maybe a bit closer to actual use cases, e.g. "Let's use twice as many signal events than background events".

I don't have a personal preference.